packages/web-components/fast-foundation/src/checkbox/checkbox.template.ts

```diff
@@ -34,9 +34,14 @@ export const CheckboxTemplate = html<Checkbox>`
                 <div part="indeterminate-indicator" class="indeterminate-indicator"></div>
             </slot>
         </div>
-        ${when(
-            x => x.childNodes.length,
-            html` <label part="label" class="label"><slot></slot></label> `
-        )}
+        <label
+            part="label"
+            class="${x =>
+                x.defaultSlottedNodes && x.defaultSlottedNodes.length
+                    ? "label"
```

Comment:
I think this should always have the label class - but additionally a 'hidden' class if it is hidden.